src/lib/core/option/option.html

```diff
@@ -1,3 +1,5 @@
+<md-pseudo-checkbox class="md-option-pseudo-checkbox" *ngIf="multiple"
```

Comment:
Good point. I did this before we did the full switch to the `mat-` prefix and had forgotten about it.